Source/Engine/Engine/RandomUtil.cs

```diff
@@ -24,5 +24,80 @@ public static float Rand()
         {
             return Random.Next(0, int.MaxValue) / (float)int.MaxValue;
         }
+
+        /// <summary>
+        ///Generates a random number within a specified range.
+        /// </summary>
+        /// <param name="min">The minimum value of the range.</param>
+        /// <param name="max">The maximum value of the range.</param>
+        /// <returns>The random number.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static float RandRange(float min, float max)
```

Comment:
It might be worth mimicking one of the existing methods for consistency :) https://github.com/FlaxEngine/FlaxEngine/blob/97ed46bc4ff36a0269b50b138c08908461ee9f96/Source/Engine/Utilities/Extensions.cs#L265

Reply:
I just remade it easier to be inside one class. 
Especially, for people who came from the other game engines.
 It's distributed  but seems I a little overcomplicated it by recreating my own version xD